python/ray/data/exceptions.py

```diff
@@ -77,11 +77,9 @@ def handle_trace(*args, **kwargs):
                     "https://github.com/ray-project/ray/issues/new/choose"
                 )
 
-            if log_to_stdout:
-                logger.exception("Full stack trace:")
```

Comment:
This is dead code. If `log_to_stdout` is `True`, we exit early: https://github.com/ray-project/ray/blob/905ca61afb1e24b876f4c8c17b78c192013b13e4/python/ray/data/exceptions.py#L55-L58